deploy-preview.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Should probably use 
```
#!/usr/bin/env bash
```

Reply:
Yeah that looks good to me. Though to be frank the current state of these scripts are more documentation than very useful out of the box. An update to the building and previewing instructions would be good as a separate issue.